src/views/InvolvementProfile/components/Membership/components/AdminCard/index.jsx

```diff
@@ -71,18 +71,15 @@ const AdminCard = ({ createSnackbar, isSiteAdmin, involvementDescription, onAddM
       onAddMember();
       setIsDialogOpen(false);
     } catch (error) {
-      if (error.Message === 'The Person is already part of the activity.') {
-        createSnackbar(`${username} is already a member`, 'info');
-      } else {
-        switch (error.name) {
-          case 'NotFoundError':
-            createSnackbar('Nobody with that username was found', 'error');
-            break;
+      switch (error.name) {
+        case 'NotFoundError':
+          createSnackbar('Nobody with that username was found', 'error');
+          break;
 
-          default:
-            console.log(error);
-            break;
-        }
+        default:
+          createSnackbar('This member could not be added', 'error');
+          console.log(error);
+          break;
```

Comment:
You don't have to break the default case since it is at the end of the switch statement. Up to you if you want to keep it or not.
```suggestion
```

Reply:
I'll remove it. Thank you!